crates/common/consensus/src/genesis.rs

```diff
@@ -0,0 +1,10 @@
+use alloy_primitives::{B256, aliases::B32};
+use serde::{Deserialize, Serialize};
+
+/// Genesis Config store.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct Genesis {
+    pub genesis_time: usize,
```

Comment:
```suggestion
    #[serde(with = "serde_utils::quoted_u64")]
    pub genesis_time: u64,
```

genesis_time should probably be a u64, as usize can be a u32 or u64 depending on the CPU archecture.

`#[serde(with = "serde_utils::quoted_u64")]` will make `json!()` print `genesis_time` as a quoted string